en/docs/deploy-operate/observe/metrics-overview.md

```
@@ -0,0 +1,191 @@
+### Prerequisites
+
+| Requirement | Details |
+|-------------|---------|
+| Prometheus | Version 2.40 or later |
+| Network | Prometheus must be able to reach the metrics endpoint |
+
+### Configure integration for Prometheus
+
+Navigate to the file explorer of the WSO2 Integrator and add the following to the `main.bal` file.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)